web/src/invite.ts

```diff
@@ -351,6 +352,11 @@ function open_invite_user_modal(e: JQuery.ClickEvent<Document, undefined>): void
 
         $("#invite-user-modal .dialog_submit_button").prop("disabled", true);
 
+        const default_stream = streams.filter((stream) => stream.default_stream);
+        for (const stream of default_stream) {
+            $(`input[value=${stream.stream_id}]`).prop("checked", true);
+        }
```

Comment:
Why are these JS changes required for a commit that appears to mainly just refactor some template code?